src/client/src/containers/ProjectNameAndOutput/index.tsx

```diff
@@ -154,8 +168,8 @@ const ProjectNameAndOutput = (props: Props) => {
         )}
         {projectNameMaxLength && (
           <div className={styles.errorMessage}>
-            {props.intl.formatMessage(messages
-              .nameTooLong as FormattedMessage.MessageDescriptor,
+            {props.intl.formatMessage(
+              messages.nameTooLong as FormattedMessage.MessageDescriptor,
```

Comment:
still don't understand why we get these phantom diffs...do some people not run prettier when they commit?

Reply:
Yeah, maybe some do not have automatic `format on save` enabled. I'll ask the team about this